src/routes/event-match.tsx

```diff
@@ -163,12 +163,11 @@ const EventMatch = ({ eventKey, matchKey }: Props) => {
   return (
     // page setup
     <Page
-      back={`/events/${eventKey}`}
       name={
-        m.group +
-        (m.num ? ' Match ' + m.num : '') +
-        ' - ' +
-        (event ? event.name : eventKey)
+        <>
+          {m.group + (m.num ? ' Match ' + m.num : '') + ' - '}
+          <a href={`/events/${eventKey}`}> {event ? event.name : eventKey} </a>
```

Comment:
We do not need these extra spaces

```suggestion
          <a href={`/events/${eventKey}`}>{event ? event.name : eventKey}</a>
```

Also, this can use optional chaining syntax:

```suggestion
          <a href={`/events/${eventKey}`}> {event?.name || eventKey} </a>
```